tests/v2/validation/rbac/rbac_test.go

```diff
@@ -62,11 +63,12 @@ func (rb *RBTestSuite) SetupSuite() {
 
 func (rb *RBTestSuite) ValidateListCluster(role string) {
 
-	//Testcase1 Verify cluster members - Owner/member are able to list clusters
+	//Testcase1 Verify cluster members - Owner/member,  Project members - Owner/member are able to list clusters
 	clusterList, err := rb.standardUserClient.Steve.SteveType(clusters.ProvisioningSteveResouceType).ListAll(&types.ListOpts{})
 	require.NoError(rb.T(), err)
 	assert.Equal(rb.T(), 1, len(clusterList.Data))
-	actualClusterID := clusterList.Data[0].Status.(interface{}).(map[string]interface{})["clusterName"]
+
+	actualClusterID := clusterList.Data[0].Status.(map[string]interface{})["clusterName"]
```

Comment:
Instead of typecasting Anu, can you convert it to the ClusterStatus object like how we discussed yesterday?